Module5/Task/Entities/Game.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Linq;
+using Task.Initializers;
+
+namespace Task.Entities
+{
+    public class Game
+    {
+        private readonly GameViewer _gameViewer;
+        private readonly DataToView _dataToView;
+
+        public Game()
+        {
+            _gameViewer = new GameViewer();
+            var trapCreator = new TrapInitializer();
+            IsGameFinished = false;
```

Comment:
IsGameFinished = false; - redundant

what value IsGameFinished  will have if you are not assign any?